doc/content/reference/telemetry/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Telemetry"
+description: ""
+aliases: [/telemetry]
+new_in_version: "3.27.1"
+---
+
+This section contains detailed information on telemetry collected by the [CLI]({{<ref "/reference/cli">}}) and by each of {{% tts %}}\'s [components]({{<ref "/components">}}).
+<!--more-->
+
+{{% tts %}}'s [CLI]({{<ref "/reference/cli">}}) and [components]({{<ref "/components">}}) collect telemetry data, which is used to help us understand how users interact with the product. A few examples are: the insight on the user's usage of organizations and a better understanding of what type of machines the CLI is more commonly used. Telemetry collection is **enabled** by default and does **not** collect personally identifiable information. The following subsections explain what information is collected and how it is anonymised.
```

Comment:
Replace 
`A few examples are: the insight on the user's usage of organizations and a better understanding of what type of machines the CLI is more commonly used.` 
with 
`For example, we gain insights on how users make use of organizations, what environment is used to run the CLI etc.`